### PR TITLE
Add support for itunes episode & season

### PR DESCRIPTION
**Pull request by saerdnaer**

### CODE REVIEW

**Comment by olleolleolle** [comment]
I was able to locate the information about Apple's podcasting RSS requirements: https://help.apple.com/itc/podcasts_connect/#/itcb54353390

File: test/test-itunes.rb [hunk 1]
```diff
@@ -99,6 +99,18 @@ def test_summary
       end
     end
 
+    def test_season
+      assert_assert_itunes_season(%w(channel)) do |content, xmlns|
```
Comment (olleolleolle) [inline test/test-itunes.rb:103]:
```suggestion
      assert_itunes_season(%w(channel)) do |content, xmlns|
```

File: test/test-itunes.rb [hunk 2]
```diff
@@ -99,6 +99,18 @@ def test_summary
...
+         make_rss20(make_channel20(content), xmlns)
+      end
+    end
+
+    def test_episode
+      assert_assert_itunes_episode(%w(channel)) do |content, xmlns|
```
Comment (olleolleolle) [inline test/test-itunes.rb:109]:
```suggestion
      assert_itunes_episode(%w(channel)) do |content, xmlns|
```

**Review by olleolleolle — CHANGES_REQUESTED** [review]
I saw two tiny typos when running the CI job.

Cool feature.

**Comment by kou** [comment]
It seems that both of `<itunes:episode>` and `<itunes:season>` should exist in `item` not `channel`.

It seems that both of them needs validation (a non-zero integer). Do you want me to take over it?

**Comment by saerdnaer** [comment]
@kou Yes thats, right. Feel free to take it over.

**Comment by kou** [comment]
I've fixed a bug that `channel` still parses `episode` and `season`.
I've also added non zero positive integer validation.

I've merged. Thanks.

**Pull request merged**